datafusion/physical-expr/src/window/partition_evaluator.rs

```diff
@@ -100,14 +99,6 @@ pub trait PartitionEvaluator: Debug + Send {
         false
     }
 
-    /// Returns the internal state of the window function
```

Comment:
The point of the PR is to get of the functions on this trait that serialized / set state (which has a single use)